src/Sulu/Component/Content/Document/Subscriber/LastModifiedSubscriber.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Document\Subscriber;
+
+use Sulu\Component\Content\Document\Behavior\AuthorBehavior;
+use Sulu\Component\Content\Document\Behavior\LocalizedAuthorBehavior;
+use Sulu\Component\DocumentManager\Event\AbstractMappingEvent;
+use Sulu\Component\DocumentManager\Event\HydrateEvent;
+use Sulu\Component\DocumentManager\Events;
+use Sulu\Component\DocumentManager\PropertyEncoder;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Handles lastModifiedEnabled and lastModified.
+ */
+class LastModifiedSubscriber implements EventSubscriberInterface
+{
+    public const LAST_MODIFIED_PROPERTY_NAME = 'lastModified';
+    public const IS_LAST_MODIFIED_PROPERTY_NAME = 'isLastModified';
```

Comment:
This will be not used and can be removed, as we just have a getter for the lastModifiedEnabled